src/main/resources/schema/moderne-organizations.graphqls

```diff
@@ -1,9 +1,4 @@
 type Query {
-    """
-    The list of organizations that a repository belongs to.
-    """
-    organizations(repository: RepositoryInput!): [Organization!]!
-
```

Comment:
I thought we were going to create a separate template repository for organizations for DX, and only that one would remove this query? Or did I misunderstand that?

Reply:
> The plan right now is to not merge this pull request, but create a new (temporary) template for DX customers to use. Once we are ready to migrate SaaS we can delete the temporary template again.